examples/request_redraw_threaded.rs

```diff
@@ -28,10 +28,9 @@ fn main() {
         control_flow.set_wait();
 
         match event {
-            Event::WindowEvent { event, .. } => match event {
-                WindowEvent::CloseRequested => control_flow.set_exit(),
-                _ => (),
-            },
+            Event::WindowEvent { event, .. } if event == WindowEvent::CloseRequested => {
```

Comment:
It seems common to handle these as:
```suggestion
            Event::WindowEvent { event: WindowEvent::CloseRequested, .. } => {
```